src/main/java/seedu/address/logic/parser/ExportCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.ExportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ExportCommandParser implements Parser<ExportCommand> {
+
+    public static final String INVALID_EXPORT_COMMAND = "Invalid command format!\n"
+            + "export: Exports the corresponding information to a text file\n"
+            + "Parameters: INFORMATION (e.g. email, phone)\n"
+            + "Example: export email";
+    /**
+     * Parses the given {@code String} of arguments in the context of the ExportCommand
+     * and returns an ExportCommand object for execution.
+     * @throws ParseException if the user input does not conform to the expected format
+     */
+    public ExportCommand parse(String args) throws ParseException {
+        String[] split = args.split(" ");
+
+        if (split.length < 2) {
```

Comment:
Perhaps can replace `1` and `2` with named constant to improve the readability?

Reply:
Will do thanks